src/library_async.js

```diff
@@ -414,6 +416,13 @@ mergeInto(LibraryManager.library, {
     //
     // JSPI implementation of Asyncify.
     //
+
+    // Stores all the exported raw Wasm functions that are wrapped with async
+    // WebAssembly.Functions.
+    asyncExports: null,
+    isAsyncExport: function(func) {
+      return Asyncify.asyncExports && Asyncify.asyncExports.has(func);
```

Comment:
Can't we assume that `instrumentWasmExports` has already been called here?

Reply:
With dynamic linking this function can be called before instrumentWasmExports is called. The test `test_asyncify_main_module_stack_switching` was failing without this.